examples/ShowPlayer.cpp

```diff
@@ -111,49 +111,52 @@ void ShowPlayer::Loop() {
  * Seek to @p seek_time in the show file
  * @param seek_time the time (in milliseconds) to seek to
  */
-ShowLoader::State ShowPlayer::SeekTo(PlaybackTime seek_time) {
+ShowLoader::State ShowPlayer::SeekTo(uint64_t seek_time) {
   // Seeking to a time before the playhead's position requires moving from the
   // beginning of the file.  This could be optimized more if this happens
   // frequently.
-  if (seek_time < m_playback_pos) {
+  if (seek_time <= m_playback_pos) {
```

Comment:
Kind of - when the first and last frame end up being the same (could be a long static look, for example), it would only be sent on the first iteration and never again.  This forces it to be loaded again.

Reply:
I noticed this while testing seeking to exactly EOF.

Reply:
Perhaps just add a comment in the code explaining this for our future selves?